modules/persistent-storage-csi-manila-dynamic-provisioning.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-manila.adoc
+
+[id="persistent-storage-csi-manila-dynamic-provisioning_{context}"]
+= Dynamically provisioning Manila CSI volumes
+
+{product-title} installs a default StorageClass.
```

Comment:
This phrase is little bit confusing for me... In OpenShift on OpenStack the default StorageClass `standard` is for Cinder, and it is created by cluster-storage-operator: https://github.com/openshift/cluster-storage-operator/blob/665c4647a815489171eb3f9eff149ae714c98e41/assets/openstack.yaml#L6
Then, there could be several optional StorageClasses for Manila, one for each share type: https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-manila-csi-plugin.md#controller-service-volume-parameters
If I understood it correctly, I suggest to write this line with something like `{product-title} installs a StorageClass for each available Manila share type.`.

Reply:
This makes more sense, updated.